test/DebugInfo/X86/xray-split-dwarf-interaction.ll

```diff
@@ -11,7 +11,7 @@
 ; RUN:     -function-sections -data-sections \
 ; RUN:     -relocation-model=pic -filetype=obj \
 ; RUN:     -generate-type-units -o - %s | \
```

Comment:
There is a bug. There should be `%t.ll` instead of `%s`. But the test fails if I fix it. I suggest to remove this test, because SPIR-V format (debug info) doesn't preserve XRay instrumentation.

Reply:
Good catch! I've removed the test.